{
  "error": [
    "  cdawmeta.io.read_cdf_depend_0s('https://cdaweb.gsfc.nasa.gov/sp_phys/data/mms/mms1/feeps/srvy/l2/electron/2015/09/mms1_feeps_srvy_l2_electron_20150901000000_v7.1.1.cdf') failed with error: get_conditional() got an unexpected keyword argument 'use_cache'\n\nTraceback (most recent call last):\n  File \"~/git/hapi/cdawmeta/cdawmeta/generators/cadence.py\", line 69, in _depend_0s_counts\n    depend_0_names = cdawmeta.io.read_cdf_depend_0s(url, logger=logger, use_cache=use_cache)\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"~/git/hapi/cdawmeta/cdawmeta/io/read_cdf.py\", line 38, in read_cdf_depend_0s\n    cdffile = open_cdf(file, logger=logger, cache_dir=cache_dir, use_cache=use_cache)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"~/git/hapi/cdawmeta/cdawmeta/io/read_cdf.py\", line 19, in open_cdf\n    info = cdawmeta.util.get_conditional(file, file_out, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nTypeError: get_conditional() got an unexpected keyword argument 'use_cache'\n",
    "  cdawmeta.io.read_cdf_depend_0s('https://cdaweb.gsfc.nasa.gov/sp_phys/data/mms/mms1/feeps/srvy/l2/electron/2025/03/mms1_feeps_srvy_l2_electron_20250327000000_v7.5.1.cdf') failed with error: get_conditional() got an unexpected keyword argument 'use_cache'\n\nTraceback (most recent call last):\n  File \"~/git/hapi/cdawmeta/cdawmeta/generators/cadence.py\", line 69, in _depend_0s_counts\n    depend_0_names = cdawmeta.io.read_cdf_depend_0s(url, logger=logger, use_cache=use_cache)\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"~/git/hapi/cdawmeta/cdawmeta/io/read_cdf.py\", line 38, in read_cdf_depend_0s\n    cdffile = open_cdf(file, logger=logger, cache_dir=cache_dir, use_cache=use_cache)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"~/git/hapi/cdawmeta/cdawmeta/io/read_cdf.py\", line 19, in open_cdf\n    info = cdawmeta.util.get_conditional(file, file_out, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nTypeError: get_conditional() got an unexpected keyword argument 'use_cache'\n"
  ]
}